src/rgw/rgw_tag_s3.cc

```diff
@@ -0,0 +1,85 @@
+#include <map>
+#include <string>
+#include <iostream>
+
+#include "include/types.h"
+
+#include "rgw_tag_s3.h"
+
+using namespace std;
+
```

Comment:
Please don't do that. It's not as bad in .cc files, but opens us up to trouble when we upgrade. It's probably better to do using std::this and using std::that.